### DISCUSSION TITLE
what is this github exactly for ?? .... just bunch of different software ? all save to download ?

### DISCUSSION THREAD

**Discussion by Heeadhunter99** [Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist#696] [General]
just curious about this github im trying to learn any computer move

**Comment by smed79** [comment]
https://github.com/Ultimate-Hosts-Blacklist/Ultimate.Hosts.Blacklist?tab=readme-ov-file#the-ultimate-hosts-blacklist---largest-unified-hosts-file-in-the-universe
